master/getting-started/kubernetes/index.md

```diff
@@ -1,41 +1,195 @@
 ---
-title: Calico for Kubernetes
+title: Quickstart for Calico on Kubernetes
 ---
 
-Calico enables networking and network policy in Kubernetes clusters across the cloud.  Calico works
-everywhere - on all major public cloud providers and private cloud as well.
 
-Calico uses a pure IP networking fabric to provide high performance networking, and its battle-tested policy engine
-enforces high-level, intent-focused network policy.  Together, Calico and Kubernetes provide a secure,
-cloud-native platform that can scale your infrastructure to hundreds of thousands of workloads.
+### Overview
 
-## Installing Calico for Kubernetes
+This quickstart gets you a single-host Kubernetes cluster with Calico
+in approximately 15 minutes. You can use this cluster for testing and
+development.
 
-There are a number of ways to install Calico and Kubernetes.  The [installation documentation](installation)
-includes links to a number of popular guides and installers which use Calico. It also
-includes information on installing Calico on a from-scratch Kubernetes cluster using either a self-hosted Kubernetes manifest,
-or by integrating Calico into your own configuration management scripts.
+To deploy a cluster suitable for production, refer to [Installation](/getting-started/kubernetes/installation/).
 
-## Using Calico with Kubernetes
```

Comment:
So, we're replacing some content here.

ISTR that we made this page the way it was because we were directing some external links to this page, and wanted a page that helped direct users to next steps they could execute when they had a cluster.

I think we need to keep that function somehow - probably by keeping the content that was here and making the quickstart its own document. Or, we need to track down the other place that link to this and link them somewhere else instead.

Reply:
I am not sure what the external documents are that point here....perhaps Mike is pointing here from his stuff? If so, it was in discussions with Mike that we determined that we should just rename the page.....actually, he wants to direct people here. Or else, let me know how we might discover the external links.

Reply:
What I can do is rename the directory to `/quickstart/` and add a redirect so that people going to `/overview/` will go to https://docs.projectcalico.org/master/getting-started/kubernetes/installation/ instead.

Reply:
I don't think that's right.

We have guides in other places that say "Once you're done installing Calico, go here to see what you can do" and it should have a landing page which has next steps, not more instructions for installing Calico.

Reply:
I'll see if I can remember / find what the links were.

Reply:
That would help....I guess I am a little lost on this one. Thanks!